stable/spring-cloud-data-flow/README.md

```diff
@@ -76,23 +76,25 @@ The following tables list the configurable parameters and their default values.
 
 ### Data Flow Server Configuration
 
-| Parameter                         | Description                                        | Default          |
-| --------------------------------- | -------------------------------------------------- | ---------------- |
-| server.version                    | The version/tag of the Data Flow server            | 2.2.0.RELEASE
-| server.imagePullPolicy            | The imagePullPolicy of the Data Flow server        | IfNotPresent
-| server.service.type               | The service type for the Data Flow server          | LoadBalancer
-| server.service.annotations        | Extra annotations for service resources            | {}
-| server.platformName               | The name of the configured platform account        | default
-| server.service.externalPort       | The external port for the Data Flow server         | 80
+| Parameter                         | Description                                                        | Default          |
+| --------------------------------- | ------------------------------------------------------------------ | ---------------- |
+| server.version                    | The version/tag of the Data Flow server                            | 2.2.0.RELEASE
+| server.imagePullPolicy            | The imagePullPolicy of the Data Flow server                        | IfNotPresent
+| server.service.type               | The service type for the Data Flow server                          | LoadBalancer
+| server.service.annotations        | Extra annotations for service resources                            | {}
+| server.platformName               | The name of the configured platform account                        | default
+| server.service.externalPort       | The external port for the Data Flow server                         | 80
+| server.configMap                  | (If set) Use existing ConfigMap for Data Flow server configuration | (empty)
```

Comment:
please update to:

`| server.configMap                  | Custom ConfigMap name for Data Flow server configuration | `

Reply:
Done